### PR TITLE
Added transition to the members card

### PR DESCRIPTION
**Pull request by musabansari-1**

### CODE REVIEW

File: components/member-card/card.module.scss [hunk 1]
```diff
@@ -13,7 +13,8 @@
   
   box-shadow: 0 0 15px -7px rgba(0, 0, 0, 0.65);
   border-radius: 5%;
-  transition: transform 0.2s ease-in-out;
+  // transition: transform 0.2s ease-in-out;
```
Comment (sumitd94) [inline components/member-card/card.module.scss:16]: Please remove this code. We do not need this.

Comment (musabansari-1) [reply components/member-card/card.module.scss:16]: the commented one?

Comment (sumitd94) [reply components/member-card/card.module.scss:16]: yes

**Comment by sumitd94** [comment]
#256

**Pull request closed without merging**